packages/browser/src/exports.ts

```diff
@@ -57,5 +57,6 @@ export {
   opera11StackLineParser,
   winjsStackLineParser,
 } from './stack-parsers';
+export { eventFromException, eventFromMessage, parseStackFrames } from './eventbuilder';
```

Comment:
Why do we need `parseStackFrames` exported?

Reply:
Originally I thought so we can patch the behavior. But actually, we can do it more simpler. So we don't need it.

Reply:
```suggestion
export { eventFromException, eventFromMessage } from './eventbuilder';
```